include/addendum.txt

```diff
@@ -0,0 +1,73 @@
+.. _addendum:
+
+Addendum
+========
+
+Currently a large process is underway to change how the middleware operates.
+This new version is the Gen3 Middleware and consists of a rewrite of Butler as
+well as the addition of SuperTasks and pre-flight. The pre-flight will use the
+`Gen3 Butler`_, as well as the `SuperTask`_ to create a **quantum graph**
+(workflow like graph) that can be executed to run the actual pipelines. This
+work is in the implementation phase, depending on the results and changes of
+this Gen3 Middleware we will re-evaluate some of the workflow systems to see
+which on fits best in this framework.
+
+During this time we have briefly looked at the Parsl software as another option
+of a workflow system as well as take the DESDM framework, which we have used in
```

Comment:
In "as well as take the DESDM framework, which we have used in operations already, and ported it to Python 3 to be ready as another solution."   either change tense of "take" to match "looked" or reword to something like "as well as successfully prototyped using the DESDM framework in LSST operations additionally porting it to Python 3 to be ready as another solution."